board_types.txt

```diff
@@ -206,6 +209,11 @@ AP_HW_CUBEBLACK_PERIPH               1401
 AP_HW_PIXRACER_PERIPH                1402
 AP_HW_SWBOOMBOARD_PERIPH             1403
 
+#IDs 5600-5699 reserved for ZeroOne
+AP_HW_ZeroOne_X6                     5600
+AP_HW_ZeroOne_PMU                    5601
+AP_HW_ZeroOne_GNSS                   5602
```

Comment:
```suggestion
AP_HW_ZeroOne_PMU                    5601
AP_HW_ZeroOne_GNSS                   5602
```

Add to AP first then document here.

Reply:
Yes, these two lines have been added to the Ardupilot.